lms/templates/learner_dashboard/empty_programs_list.underscore

```diff
@@ -1,9 +1,8 @@
-
-    <section class="empty-programs-message">
-        <p><%- gettext('You are not enrolled in any XSeries Programs yet.') %></p>
-            <a class="find-xseries-programs" href="<%- xseriesUrl %>">
-                <i class="action-xseries-icon" aria-hidden="true"></i>
-                <span><%- gettext('Explore XSeries Programs') %></span>
-            </a>
-    </section>
+<section class="empty-programs-message">
+    <h2 class="text"><%- gettext('You are not enrolled in any XSeries Programs yet.') %></h2>
+    <a class="find-xseries-programs" href="<%- xseriesUrl %>">
+        <span class="action-xseries-icon" aria-hidden="true"></span>
+        <span><%- gettext('Explore XSeries Programs') %></span>
```

Comment:
Can we move this request to a separate ticket (saying as the references are already present in the code base)? Unless @marcotuts is comfortable with removing all references to XSeries it seems like we will need to utilize theming.